apps/desktop/src/shared/context/contributors/githubPr.ts

```
@@ -0,0 +1,41 @@
+}
+
+export const githubPrContributor: ContextContributor<{
+	kind: "github-pr";
+	url: string;
+}> = {
+	kind: "github-pr",
+	displayName: "GitHub Pull Request",
+	description: "Full PR metadata fetched and inlined as context.",
+	requiresQuery: true,
+	async resolve(source, ctx) {
+		let pr;
```

Biome (2.4.11)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)